plugins/flow-visibility/clickhouse-monitor/main.go

```diff
@@ -17,7 +17,10 @@ package main
 import (
```

Comment:
same comment as above; are the monitor changes related to the PersistentVolume support?

Reply:
Most of the monitor changes are related to the PV support. As with PV, user may store ClickHouse data together with other software, in which way the disk usage percentage does not indicate the actual space usage percentage of the ClickHouse. thus we need some update in the disk usage checking logic.

The only unrelated change is I move `deletePercentage` and `threshold` parameter from hard-coded ones to env variables in yaml to make customized value more convenient. I could take out it from this PR.

Reply:
Removed some unrelated changes.